src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/HttpClientBuilderExtensions.StandardResilience.cs

```diff
@@ -37,16 +36,16 @@ public static IHttpStandardResiliencePipelineBuilder AddStandardResilienceHandle
     }
 
     /// <summary>
-    /// Adds a <see cref="PolicyHttpMessageHandler" /> that uses a standard resilience pipeline with default options to send the requests and handle any transient errors.
-    /// The pipeline combines multiple policies that are configured based on HTTP-specific <see cref="HttpStandardResilienceOptions"/> options with recommended defaults.
+    /// Adds a standard resilience handler that uses a multiple resilience strategies with default options to send the requests and handle any transient errors.
+    /// The resilience strategy combines multiple strategies that are configured based on HTTP-specific <see cref="HttpStandardResilienceOptions"/> options with recommended defaults.
```

Comment:
Only one sentence in the summary please. Other functions in this file also have this problem.

Reply:
Done.